lib/LinkEmbedder/Link.pm

```diff
@@ -382,9 +383,13 @@ __DATA__
 <a class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" href="<%= $l->url %>"><%= Mojo::Util::url_unescape($l->url) %></a>
 % }
 @@ video.html.ep
-<video class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" height="640" width="480" preload="metadata" controls>
-% for my $s (@{$l->{sources} || []}) {
-  <source src="<%= $s->{url} %>" type="<%= $s->{type} || '' %>">
-% }
-  <p>Your browser does not support the video tag.</p>
-</video>
+@@ video.html.ep
+<div class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" height="640" width="480" preload="metadata" controls>
```

Comment:
The div should not have height/width/preload/controls attributes.

Reply:
Ok